eth/beacon/types/validator_record.py

```diff
@@ -0,0 +1,54 @@
+import rlp
+
+from eth.constants import (
+    ZERO_ADDRESS,
+    ZERO_HASH32,
+)
+from eth.rlp.sedes import (
+    address,
+    int16,
+    int64,
+    int128,
+    int256,
+    hash32,
+)
+
+
+class ValidatorRecord(rlp.Serializable):
+    """
+    Note: using RLP until we have standardized serialization format.
+    """
+    fields = [
+        # The validator's public key
+        ('pubkey', int256),
+        # What shard the validator's balance will be sent to after withdrawal
+        ('withdrawal_shard', int16),
+        # And what address
+        ('withdrawal_address', address),
+        # The validator's current RANDAO beacon commitment
+        ('randao_commitment', hash32),
+        # Current balance
+        ('balance', int128),
+        # Dynasty where the validator is inducted
+        ('start_dynasty', int64),
+        # Dynasty where the validator leaves
+        ('end_dynasty', int64),
+    ]
+
+    def __init__(self,
+                 pubkey=b'',
```

Comment:
Is the `0` public key a good reasonable default?

Reply:
will remove all default values from ValidatorRecord.